### PR TITLE
Change criteria

### PR DESCRIPTION
**Pull request by aman-tunerlabs**

### CODE REVIEW

File: controllers/v1/reportsController.js [hunk 1]
```diff
@@ -2598,4 +2575,51 @@ module.exports = class Reports {
     })
     return schoolFieldArray;
   }
+
+  evaluationFrameworkDocument(evaluationFrameworksDocuments) {
```
Comment (aks30) [inline controllers/v1/reportsController.js]: Please move this to util file for common use. Also, try to achieve this in one function. @aman-tunerlabs

Comment (aman-tunerlabs) [reply controllers/v1/reportsController.js]: Added functionality in utils. EvaluationFrameworkDocuments will be refering to generatePathToCriteria in utils.

File: controllers/v1/reportsController.js [hunk 2]
```diff
@@ -868,61 +911,25 @@ module.exports = class Reports {
...
-                input.push(criteriaReportObject);
-              }
-            });
+        submissionDocument.criterias && submissionDocument.criterias.forEach(submissionCriterias => {
+          let levels = Object.values(submissionCriterias.rubric.levels);
+
+          if (submissionCriterias._id) {
+            let criteriaReportObject = {
+              "Path To Criteria": arr[submissionCriterias._id.toString()] ? arr[submissionCriterias._id.toString()].parentPath : "",
+              "Level 1": levels.find(level => level.level == "L1").description,
```
Comment (aks30) [inline controllers/v1/reportsController.js]: Please can we make this dynamic @aman-tunerlabs

Comment (aman-tunerlabs) [reply controllers/v1/reportsController.js]: Done .

**Pull request merged**